cmake/python.cmake

```diff
@@ -31,7 +31,12 @@ if(${SWIG_VERSION} VERSION_GREATER_EQUAL 4)
   list(APPEND CMAKE_SWIG_FLAGS "-doxygen")
 endif()
 
-if(UNIX AND NOT APPLE)
+# On OpenBSD, int64_t is long long int
+if(CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
+  list(APPEND CMAKE_SWIG_FLAGS "-DSMALL_LONG")
+endif()
+
+if(UNIX AND NOT APPLE AND NOT (CMAKE_SYSTEM_NAME STREQUAL "OpenBSD"))
```

Comment:
The parens around `CMAKE_SYSTEM_NAME STREQUAL "OpenBSD"` may be superfluous. I'm not familiar with operator associativity rules for CMake files.

Reply:
like as it, better readability...